body_frames:
  base_link:
    imu:
      translation:
        x: 0.0
        y: 0.0
        y: 0.0
      rotation:
        w: 1.0
        x: 0.0
        y: 0.0
        z: 0.0
    lidar:
      translation:
        x: 0.0
        y: 0.0
        z: 0.0
      rotation:
        w: 1.0
        x: 0.0
        y: 0.0
        z: 0.0
    stereo:
      translation:
        x: 0.0
        y: 0.0
        z: 0.0
      rotation:
        w: 1.0
        x: 0.0
        y: 0.0
        z: 0.0

# Stereo Camera
stereo:
  baseline: 0.1
